src/PresentationalComponents/TagsToolbar/ManageTags.js

```diff
@@ -1,36 +1,113 @@
-import { Modal } from '@patternfly/react-core/dist/js/components/Modal/Modal';
+import React, { useEffect, useState } from 'react';
+
+import { DEBOUNCE_DELAY } from '../../AppConstants';
 import PropTypes from 'prop-types';
-import React from 'react';
-import TagsList from './TagsList';
+import { TagModal } from '@redhat-cloud-services/frontend-components/components/TagModal';
 import { connect } from 'react-redux';
+import debounce from '../../Utilities/Debounce';
 import { injectIntl } from 'react-intl';
```

Comment:
i know i know, not related to the work.. but lets use `import { useIntl } from 'react-intl';` here and not pass it through the props (means we can cleanup it on the props).... checkout  src/export/systemspdf.js for an exmaple